content/zh/guide/v10/getting-started.md

```diff
@@ -252,3 +236,24 @@ export default {
   }
 }
 ```
+
+此外，您可能需要启用 `skipLibCheck`,就像我们在上面的示例中所做的那样。一些 React 库使用的类型可能没有在`preact/compat`中提供（尽管我们会尽力修复这些问题），因此这些库可能会导致 TypeScript 编译错误。通过设置 `skipLibCheck`, 您可以告诉 TypeScript 不需要对所有
+`.d.ts` 文件（通常是 `node_modules` 中的库文件）进行完整检查，从而解决这些错误。
+
+#### 使用导入映射（Import Maps）进行别名配置
+
+```html
+<script type="importmap">
+  {
+    "imports": {
+      "preact": "https://esm.sh/preact@10.23.1",
+      "preact/": "https://esm.sh/preact@10.23.1/",
+      "react": "https://esm.sh/preact@10.23.1/compat",
+      "react/": "https://esm.sh/preact@10.23.1/compat/",
+      "react-dom": "https://esm.sh/preact@10.23.1/compat",
+    }
+  }
+</script>
+```
+
+同样也可以参考[Import Maps -- Recipes and Common Patterns](/guide/v10/no-build-workflows#recipes-and-common-patterns)作为例子.
```

Comment:
The fragment link (`#...`) is unfortunately not language-agnostic, as the ID is generated from the header content itself (so `# Recipes and Common Patterns` turns into `recipes-and-common-patterns`, etc). Not a huge deal by any means, and I'd like to change that, but these links will need the "translated" IDs.

```suggestion
同样也可以参考[Import Maps -- Recipes and Common Patterns](/guide/v10/no-build-workflows#示例与常见模式)作为例子.
```